package.json

```diff
@@ -27,14 +27,15 @@
     "appium-support": "^2.0.10",
     "babel-runtime": "=5.8.24",
```

Comment:
at some point of time we'll need to take care about updating this one

Reply:
Yup. But it requires re-writing all the transpiling code, since Babel 5 -> 6 was a complete change. :( It's on my plate.